nservicebus/messaging/delayed-delivery.md

```diff
@@ -26,6 +26,7 @@ Calling `bus.Defer(...)` the handling of the message is deferred by certain amou
 
 Using this mechanism delayed delivery can be achieved by calling `bus.Defer(...)` which can be seen in [Delayed Delivery Sample](/samples/delayed-delivery).
 
+Note: Similar to `bus.SendLocal`, `bus.Defer` will also change the message's reply-to address to the endpoint deferring the message. Calling `bus.Reply` on a deferred message will send the reply to itself.
```

Comment:
Bus.Defer is no longer present in v6 but users can still do

```
 var options = new SendOptions();

options.RouteToLocalEndpoint();
options.DelayDeliveryWith(TimeSpan.FromSeconds(30));

context.Send(options, message)

```

and then run into the same issue?